### PR TITLE
Placeholder changelog for release on 2022-05-09

### PR DESCRIPTION
**Pull request by bitdivine**
# Motivation
Releases should be recorded in the CHANGELOG file.

## Details
Commit:  497fb82af0ac02b4572fd011521cd481696c168d
CI:  https://github.com/dfinity/nns-dapp/actions/runs/2292792271
SHA:
  - CI: 67166f3b6d350762036446fb8c464e97bafe1a7cdd436743c4493e9518572b26
  - Max: 67166f3b6d350762036446fb8c464e97bafe1a7cdd436743c4493e9518572b26  nns-dapp.wasm
  - Maria: 67166f3b6d350762036446fb8c464e97bafe1a7cdd436743c4493e9518572b26 nns-dapp.wasm

# Changes
* Update the CHANGELOG entry.

# Tests
- [x] Rollback
- [ ] Compare mainnet config
  - [x] Svelte
  - [x] Flutter
  - [ ] Deploy testnets with various REDIRECT_TO_LEGACY values but prod in particular

### CODE REVIEW

File: release/PROPOSAL.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Upgrade frontend NNS Dapp canister to commit `497fb82af0ac02b4572fd011521cd481696c168d`
+Wasm sha256 hash: `67166f3b6d350762036446fb8c464e97bafe1a7cdd436743c4493e9518572b26` (`https://github.com/dfinity/nns-dapp/actions/runs/2292792271`)
+
+## Change Log:
+
+* Render 32 byte hashes in proposals as strings rather than byte arrays
+* Refactor the configuration to suport more testing scenarios
+* Hide "remove hotkeys" if the user is not the controller
```
Comment (lmuntanerDfinity) [inline release/PROPOSAL.md:8]: This is for Svelte, I would not add it here. The user won't see this change.

**Review by lmuntanerDfinity — APPROVED** [review]
LGTM

**Review by lmuntanerDfinity — CHANGES_REQUESTED** [review]
Sorry, one change requested

File: release/PROPOSAL.md (hunk 1 above)
Comment (bitdivine) [reply release/PROPOSAL.md:9]: Ok, removing.  Thank you for the review!

**Review by lmuntanerDfinity — APPROVED** [review]

**Pull request merged**